webapi/src/main/scala/org/knora/webapi/messages/util/search/gravsearch/transformers/ConstructToConstructTransformer.scala

```diff
@@ -7,40 +7,74 @@ package org.knora.webapi.messages.util.search.gravsearch.transformers
 
 import zio._
 
+import org.knora.webapi.messages.OntologyConstants
 import org.knora.webapi.messages.SmartIri
-import org.knora.webapi.messages.StringFormatter
 import org.knora.webapi.messages.util.search._
+import org.knora.webapi.slice.resourceinfo.domain.IriConverter
 
-import SparqlTransformer._
-
-class ConstructToConstructTransformer(
+final case class ConstructToConstructTransformer(
   sparqlTransformerLive: SparqlTransformerLive,
-  implicit val stringFormatter: StringFormatter
-) extends WhereTransformer {
+  iriConverter: IriConverter
+) {
+
+  private def transformLuceneQueryPattern(
+    luceneQueryPattern: LuceneQueryPattern
```

Comment:
I would call the argument `queryPattern" or just `pattern`, because as a rule of thumb, there is no sense to repeat the type and It will keep the code lines shorter.

Reply:
you're right. Done.